cmd/kubectl-gadget/trace/mount.go

```diff
@@ -15,52 +15,92 @@
 package trace
 
 import (
-	"encoding/json"
 	"fmt"
-	"os"
 	"strings"
 
 	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
 	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/mountsnoop/types"
-	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 
 	"github.com/spf13/cobra"
 )
 
-var mountsnoopCmd = &cobra.Command{
-	Use:   "mount",
-	Short: "Trace mount and umount system calls",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		// print header
-		switch params.OutputMode {
-		case utils.OutputModeCustomColumns:
-			fmt.Println(getCustomMountsnoopColsHeader(params.CustomColumns))
-		case utils.OutputModeColumns:
-			fmt.Printf("%-16s %-16s %-16s %-16s %-16s %-6s %-6s %-10s %s\n",
-				"NODE", "NAMESPACE", "POD", "CONTAINER",
-				"COMM", "PID", "TID", "MNT_NS", "CALL")
-		}
+type MountParser struct {
+	BaseTraceParser
+}
 
-		config := &utils.TraceConfig{
-			GadgetName:       "mountsnoop",
-			Operation:        "start",
-			TraceOutputMode:  "Stream",
-			TraceOutputState: "Started",
-			CommonFlags:      &params,
-		}
+func newMountCmd() *cobra.Command {
+	commonFlags := &utils.CommonFlags{
+		OutputConfig: utils.OutputConfig{
+			// The columns that will be used in case the user does not specify
+			// which specific columns they want to print.
+			CustomColumns: []string{
+				"node",
+				"namespace",
+				"pod",
+				"container",
+				"pid",
+				"tid",
+				"mnt_ns",
+				"comm",
+				"op",
+				"ret",
+				"lat",
+				"fs",
+				"src",
+				"target",
+				"data",
+				"call",
+				"flags",
+			},
```

Comment:
I think it's too much for the default arguments. Any reason no to keep using the same as before? https://github.com/kinvolk/inspektor-gadget/blob/08d49ba693a7c734993afe32f0ee7af797a83682/cmd/kubectl-gadget/trace/mount.go#L39-L41

Reply:
Oops, my fault. I added the full list of fields and I didn't take into account that not all of them were printed. I will update the PR to continue printing the same list of columns. 

The other changes in the integration-tests was because I changed the order of the columns in some gadgets because I was trying to keep the same order in all of them: node, namespace, pod, container, pid, comm (if available) and the rest.